app/src/main/java/io/gnosis/safe/ui/dialogs/EnsInputDialog.kt

```diff
@@ -16,6 +16,8 @@ import io.gnosis.safe.R
 import io.gnosis.safe.di.components.DaggerViewComponent
 import io.gnosis.safe.di.modules.ViewModule
 import io.gnosis.safe.helpers.AddressHelper
+import io.gnosis.safe.ui.transaction.TransactionsViewModel.Companion.OPACITY_FULL
```

Comment:
better to extract those values and put them into dimens.xml

Reply:
What would be the advantage to have this in dimens.xml? It is not used in layout.

Reply:
it is used in EnsInputDialog. Which is strange. Why should EnsInputDialog import something from TransactionsViewModel?

Reply:
How can put floating point values into diments.xml?

Reply:
```<item name="name" format="float" type="dimen">1.0</item>```

Reply:
```kotlin

Reply:
That is a lot of code. Maybe I just revert to 1F and 0.5F?